### DISCUSSION TITLE
hello, world!

### DISCUSSION THREAD

**Discussion by henrytill** [musnix/musnix#159] [Announcements]
Feel free to use this to discuss musnix-related stuff.

**Comment by smoge** [comment]
Hello! Are there many of us in the world? Let's be in touch.

**Comment by magnetophon** [comment]
:wave:

**Comment by torgeir** [comment]
Hello! :wave: Just checking this out for a realtime enabled workstation to play :guitar:!

Browsing through the source, I discovered [this](https://github.com/musnix/musnix/blob/55b46a6a116f8b6cee17e05e85fadfd23865327a/modules/base.nix#L56) - should it not be `threadirqs`? (search [here](https://docs.kernel.org/admin-guide/kernel-parameters.html)) - or does that not make a difference?

**Reply by henrytill** [reply]
Indeed, that looks incorrect - thanks for pointing that out.  

Feel free to drop a PR, otherwise I will fix it sometime this week.
